TheMovieDb/Modules/MainTab/MainTabWireFrame.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MainTabWireFrame.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import Foundation
+import UIKit
+
+class MainTabWireFrame: MainTabWireFrameProtocol {
+    class func createMainTabModule() -> UIViewController {
```

Comment:
are this a class func method ? if so why a class and not static instead ?

Reply:
😅 now are static func